### ISSUE TITLE
Core threads must have nonzero keep alive times 

### ISSUE THREAD

**Issue by giniedp** [adjust/android_sdk#225]
Since Adjust version 4.10.0 the following error occurs

```
Caused by: java.lang.IllegalArgumentException: Core threads must have nonzero keep alive times
   at java.util.concurrent.ThreadPoolExecutor.allowCoreThreadTimeOut(ThreadPoolExecutor.java:1608)
   at com.adjust.sdk.CustomScheduledExecutor.<init>(CustomScheduledExecutor.java:47)
   at com.adjust.sdk.ActivityHandler.<init>(ActivityHandler.java:165)
   at com.adjust.sdk.ActivityHandler.getInstance(ActivityHandler.java:222)
   at com.adjust.sdk.AdjustInstance.onCreate(AdjustInstance.java:31)
   at com.adjust.sdk.Adjust.onCreate(Adjust.java:36)
```

I have tested compileSdkLevel 23 and 24 and the following Android versions
- Error occurs on API Level 16
- Error occurs on API Level 19
- Error occurs on API Level 21
- Error does not occur on API Level 23

The error does not occur on Adjust version 4.7.0

**Comment by nonelse** [comment]
Thanks.
We found that by default API level 23 and over has the default keep alive time of 10 milliseconds, but that is not the case under.
We'll make a release with an explicit keep alive time to prevent this error.

**Comment by nonelse** [comment]
Fixed with release [v4.10.1](https://github.com/adjust/android_sdk/releases/tag/v4.10.1), thanks!
